packages/longhorn/package.yaml

```diff
@@ -0,0 +1,8 @@
+url: https://github.com/longhorn/charts.git
+subdirectory: charts/longhorn
+type: git
+commit: 0a88eaae4d7bdce762af942ea490d72aee338bb1
+packageVersion: 00
+generateCRDChart:
+  enabled: true
+  providesGVR: longhorn.io/v1beta1
```

Comment:
Can you remove this, it doesn't have an effect anymore. The annotation on chart is the correct way and it already is there.